x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/CrossClusterAccessAuthenticationService.java

```diff
@@ -94,18 +93,6 @@ public void authenticate(final String action, final TransportRequest request, fi
         }
 
         // This check is to ensure all nodes understand cross_cluster_access subject type
```

Comment:
I don't think this comment applies anymore.

Reply:
Removed.